code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
Interesting. That is the same code that I had, and I was still getting the same error. In fact, I am still getting the same error with your branch. Perhaps we are testing it in different ways.

I am going to app lab, adding a button, and then clicking on the color picker button, at which point the error shows up. How are you testing this?